components/Cards/src/Theming/MDCCard+MaterialTheming.m

```diff
@@ -46,10 +44,7 @@ - (void)applyThemeWithScheme:(id<MDCContainerScheming>)scheme {
 }
 
 - (void)applyThemeWithColorScheme:(id<MDCColorScheming>)colorScheme {
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-  [MDCCardsColorThemer applySemanticColorScheme:colorScheme toCard:self];
-#pragma clang diagnostic pop
+  self.backgroundColor = colorScheme.surfaceColor;
```

Comment:
Tip: Nit: Consider removing this method and just applying the surface color above. It would make the code easier to read.

Reply:
Done.